modules/rosapps/applications/screensavers/mazescr/lang/ro-RO.rc

```diff
@@ -1,47 +1,53 @@
-/* Translator: Ștefan Fulea (stefan dot fulea at mail dot com) */
+ /*
+ * PROJECT:     Maze ScreenSaver
+ * LICENSE:     HPND (https://spdx.org/licenses/HPND)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2024 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_DEFAULT
 
 IDD_DLG_SCREEN DIALOGEX DISCARDABLE  0, 0, 276, 112
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU
-CAPTION "Configurație animație de inactivitate „Labirint”"
+CAPTION "Setări pentru economizorul de ecran Labirint"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT           "Mărime labirint:",IDC_STATIC,7,7,90,12,SS_CENTERIMAGE
+    LTEXT           "Mărimea Labirintului:",IDC_STATIC,7,7,90,12,SS_CENTERIMAGE
     CONTROL         "Slider1",IDC_SLIDER_SIZE,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,101,7,140,12
     CTEXT           "Static",IDC_TEXT_SIZE,245,7,24,12,SS_CENTERIMAGE,WS_EX_STATICEDGE
-    LTEXT           "Timp ante-rezolvare:",IDC_STATIC,7,28,90,12,SS_CENTERIMAGE
+    LTEXT           "Întârziere de pre-rezolvare:",IDC_STATIC,7,28,90,12,SS_CENTERIMAGE
     CONTROL         "Slider2",IDC_SLIDER_PRESD,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,101,28,140,12
     CTEXT           "Static",IDC_TEXT_PRESD,245,28,24,12,SS_CENTERIMAGE,WS_EX_STATICEDGE
-    LTEXT           "Timp post-rezolvare:",IDC_STATIC,7,49,90,12,SS_CENTERIMAGE
+    LTEXT           "Întârziere de post-rezolvare:",IDC_STATIC,7,49,90,12,SS_CENTERIMAGE
     CONTROL         "Slider3",IDC_SLIDER_POSTSD,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,101,49,140,12
     CTEXT           "Static",IDC_TEXT_POSTSD,245,49,24,12,SS_CENTERIMAGE,WS_EX_STATICEDGE
-    LTEXT           "Timp de rezolvare:",IDC_STATIC,7,70,90,12,SS_CENTERIMAGE
+    LTEXT           "Întârziere rezolvare:",IDC_STATIC,7,70,90,12,SS_CENTERIMAGE
     CONTROL         "Slider4",IDC_SLIDER_SD,"msctls_trackbar32",TBS_BOTH | TBS_NOTICKS | WS_TABSTOP,101,70,140,12
     CTEXT           "Static",IDC_TEXT_SD,245,70,24,12,SS_CENTERIMAGE,WS_EX_STATICEDGE
-    DEFPUSHBUTTON   "Con&firmă",IDOK,59,91,50,14
-    PUSHBUTTON      "A&nulează",IDCANCEL,113,91,50,14
-    PUSHBUTTON      "&Despre",IDABOUT,167,91,50,14
+    DEFPUSHBUTTON   "OK",IDOK,59,91,50,14
+    PUSHBUTTON      "Revocare",IDCANCEL,113,91,50,14
+    PUSHBUTTON      "Despre",IDABOUT,167,91,50,14
```

Comment:
You stripped the accelerator here. Prove with a pic that romanian 2k3sp2 does not have an accel here. Contrary, if it has, then add one back.

Reply:
I will add the accelerator back. I mistakenly stripped it.

Reply:
Fixed!

Reply:
Resolved.